### PR TITLE
[encointer] upgrade to polkadot v1.6.0

### PR DESCRIPTION
**Pull request by brenzi**
based on https://github.com/polkadot-fellows/runtimes/pull/159

Encointer pallets `5.x.x` all depend on polkadot-v1.6.0 crates.io releases
(No changes of pallet functionality) 
see: https://github.com/encointer/pallets/pull/370

_hint: this version of encointer pallets is compatible with polkadot-v1.7.0 as none of the deps have been bumped upstream_

- [x] Does not require a CHANGELOG entry (if merged together with #159)

### CODE REVIEW

**Comment by acatangiu** [comment]
I suggest we merge 159 then this one, squashing them together just creates an even bigger, harder to review PR

File: system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs [hunk 1]
```diff
@@ -1389,7 +1389,7 @@ impl_runtime_apis! {
 					let fee_index = if assets.get(0).unwrap().eq(&fee_asset) { 0 } else { 1 };
 
 					// verify transferred successfully
-					let verify = Box::new(move || {
+					let verify = Box::new(move || {let verify = Box::new(move || {
```
Comment (acatangiu) [inline system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs:1392]: accidental duplicate paste

**Review by acatangiu — APPROVED** [review]
actually the encointer diff is pretty small, we can merge it with #159

**Comment by brenzi** [comment]
I requested a pull into #159 over at @bkontur 's repo: https://github.com/bkontur/runtimes/pull/2

**Comment by bkontur** [comment]
> I requested a pull into #159 over at @bkontur 's repo: [bkontur#2](https://github.com/bkontur/runtimes/pull/2)

ok, cool, I am busy with other stuff now, but I have this PR in my backlog, don't worry :)

**Comment by bkontur** [comment]
@brenzi I will merge Encointer to 1.6 as a https://github.com/bkontur/runtimes/pull/4 where I took your PR: https://github.com/bkontur/runtimes/pull/2
so you can close this and start preparing the same bump for 1.7 :) https://github.com/polkadot-fellows/runtimes/pull/187

**Pull request closed without merging**